src/components/how/How.js

```diff
@@ -48,15 +52,18 @@ const How = () => {
             <StepText>{step2}</StepText>
           </StepGroupTruck>
           <StepGroupStars>
-            <Stars/>
+            <Stars />
             <StepText>{step4}</StepText>
           </StepGroupStars>
-          <Arrow top='136px' left='27.82%' deg='0deg'/>
-          <Arrow top='136px' left='69.35%' deg='90deg'/>
-          <Arrow top='480px' left='69.35%' deg='180deg'/>
-          <Arrow top='480px' left='27.82%' deg='270deg'/>
+          <Arrow top='136px' left='27.82%' deg='0deg' />
+          <Arrow top='136px' left='69.35%' deg='90deg' />
+          <Arrow top='480px' left='69.35%' deg='180deg' />
+          <Arrow top='480px' left='27.82%' deg='270deg' />
         </StepsWrapper>
-        <WorkButton>{buttonText}</WorkButton>
+        <WorkButton
+          onClick={openPopup}>
```

Comment:
same comment as header.js. if you put onClick={props.openPopup} you don't need the duplicate function at the top of this file